python/pyspark/sql/dataframe.py

```diff
@@ -730,8 +730,9 @@ def join(self, other, on=None, how=None):
             a join expression (Column), or a list of Columns.
             If `on` is a string or a list of strings indicating the name of the join column(s),
             the column(s) must exist on both sides, and this performs an equi-join.
-        :param how: str, default 'inner'.
-            One of `inner`, `outer`, `left_outer`, `right_outer`, `leftsemi`.
+        :param how: str, default ``inner``. Must be any of: ``inner``, ``cross``, ``outer``,
```

Comment:
I'm not really sure I agree with either or those. It's concise because both "must be one of" and "must be any of" are the same number of characters and words. It's consistent because that's what I used everywhere and there are virtually no instances of "must be one of" in Spark except in some random examples and docs.

With all that being said, I guess it is a bit strange grammatically and I am happy to change it.